### PR TITLE
✨ MSA-13876: Make host and port configurable.

### PR DESCRIPTION
**Pull request by Vignaudo**

### CODE REVIEW

File: msa_sdk/msa_api.py [hunk 1]
```diff
@@ -21,7 +22,13 @@ def host_port():
     Hostname and Port
 
     """
-    return('localhost', '8480')
+    host = 'localhost'
+    if ('UBIQUBE_MSA_HOST' in os.environ):
+        host = os.environ['UBIQUBE_MSA_HOST']
```
Comment (yguubiqube) [inline msa_sdk/msa_api.py:27]:
```suggestion
    if ('UBIQUBE_MSA_HOST' in os.environ):
        host = os.environ['UBIQUBE_MSA_HOST']
    else
        host = 'localhost'
```

File: msa_sdk/msa_api.py [hunk 2]
```diff
@@ -21,7 +22,13 @@ def host_port():
...
+    port = '8480'
+    if ('UBIQUBE_MSA_PORT' in os.environ):
+        port = os.environ['UBIQUBE_MSA_PORT']
```
Comment (yguubiqube) [inline msa_sdk/msa_api.py:30]:
```suggestion
    if ('UBIQUBE_MSA_PORT' in os.environ):
        port = os.environ['UBIQUBE_MSA_PORT']
    else
        port = '8480'
```

**Review by yguubiqube — CHANGES_REQUESTED** [review]

File: msa_sdk/msa_api.py (hunk 1 above)
Comment (fambon) [reply msa_sdk/msa_api.py:27]: or:
```
host = os.environ.get('UBIQUBE_MSA_HOST') or "localhost"
```

Comment (Vignaudo) [reply msa_sdk/msa_api.py]: @fambon ,
Thank you, it's exactly how I want to write those things !

**Review by sneUbiqube — APPROVED** [review]

**Review by UBiqube-ydu — APPROVED** [review]

**Pull request merged**